src/energyData.js

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs'
+import * as aq from 'arquero'
+import clm from 'country-locale-map'
+const { op } = aq
+
+//  Load energy data
+const pathToCsv = './data/energyData.csv?raw'
+const csvData = fs.readFileSync(pathToCsv, 'utf8')
+
+const energyRaw = aq.fromCSV(csvData)
+
+// Filter energy data to keep relevant columns
+const energyClean = energyRaw.select({
+    country: 'country',
+    year: 'year',
+    iso_code: 'iso_code',
+    gdp: 'gdp',
+    population: 'population',
+    renewables_share_energy: 'renewablesShareCon',
+    renewables_share_elec: 'renewablesShareGen',
+  })
+  .derive({ gdpPerCap : d => d.gdp/d.population})
+  .filter(
+    (d) =>
+      op.abs(d.gdp) >= 0 &&
```

Comment:
By using op.abs(), you're already getting the absolute value of columns so what this filter action is actually doing is checking whether op.abs works
```suggestion
      d.gdp >= 0 &&
```